src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
+
+type Colors = 'red' | 'green' | 'blue';
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: Colors;
+  getArea(): number;
+}
+
+function validateTriangleLength(...shapeSide: number[]): boolean {
+  const sumSides = shapeSide
+    .reduce((sideOne: number, sideTwo: number) => sideOne + sideTwo);
 
+  return shapeSide.some((side: number) => side >= (sumSides - side));
 }
 
-export class Triangle {
+function validateLength(...lengths : number[]): boolean {
+  return Math.min(...lengths) <= 0;
+}
 
+function helperArea(area: number): number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
```

Comment:
```suggestion
  readonly shape: Shape = Shape.Triangle;
```
The shape prop can be protected from changes.